doc/docs/download_cfd.md

```diff
@@ -3,28 +3,37 @@
 Download nRF Connect for Desktop's executable from Nordic Semiconductor's [website](https://www.nordicsemi.com/Software-and-Tools/Development-Tools/nRF-Connect-for-desktop).
 
 All nRF Connect for Desktop applications require the launcher v4.1.0 or later.
+See also the list of [Supported operating systems](./os_support.md).
 
 After installing and starting the launcher, you will see the list of available apps.
 See the [Launcher overview](overview_cfd.md) and [Installing nRF Connect for Desktop apps](installing_apps.md) pages for more information.
 
 ## Additional requirements
 
-Running nRF Connect for Desktop on macOS and Linux has additional requirements.
+Running nRF Connect for Desktop has the following additional requirements.
 
-### J-Link driver on macOS and Linux
+### SEGGER J-Link driver
 
-You must install SEGGER J-Link driver separately on macOS and Linux. To do so, download and
-install it from [SEGGER](https://www.segger.com/downloads/jlink) under the
-section _J-Link Software and Documentation Pack_.
+Required on all platforms.
 
-If you don't install the J-Link driver, running the apps will fail and you will get error messages
-`CouldNotFindJprogDLL`, `CouldNotOpenDLL`, `JLINKARM_DLL_NOT_FOUND`, or similar.
+Download the installer for your platform from [SEGGER J-Link Software](https://www.segger.com/downloads/jlink/#J-LinkSoftwareAndDocumentationPack).
+If you don't install the J-Link driver, running the apps will fail and you will get error messages `CouldNotFindJprogDLL`, `CouldNotOpenDLL`, `JLINKARM_DLL_NOT_FOUND`, or similar.
 
 ### udev rules for Linux only
 
 In order to access Nordic USB devices with correct permissions, you must set up _udev_ rules.
-To do so, follow the instructions in the [nrf-udev](https://github.com/NordicSemiconductor/nrf-udev)
-repository. You need to set up these rules only once.
+You need to set up these rules only once.
+
+nrf-udev can be installed by downloading a DEB file from the [nrf-udev](https://github.com/NordicSemiconductor/nrf-udev) project repository.
+
+Complete the following steps:
+
+1. Download the latest DEB file from [nrf-udev](https://github.com/NordicSemiconductor/nrf-udev).
+2. Run the following command to install nrf-udev:
+
+    ```
+    sudo dpkg -i nrf-udev_1.0.1-all.deb
+    ```
```

Comment:
@KievDevel , can you confirm libusb is not needed for Connect for Desktop? I'm mirroring https://docs.nordicsemi.com/bundle/nrfutil/page/guides/installing.html#prerequisites here.

Reply:
Short answer: I can't confirm. I assume 98% libusb is required. Though on Ubuntu it comes with OS.

Long answer: nrf-udev are required on linux (Ubuntu) to work properly with nordic devices. Otherwise some devices are not accessible. It is solving LIBUSB_ERROR_ACCESS so I assume both libraries are must.

Finally, was able to fight locks and remove libusb from VM which caused apps not being able to create nrfutil-sandbox and unable to load DLLs. So usage of the apps was limited to none.

Only thing we might mention as done on nrfutil page that "libusb-1.0 usually comes installed with Ubuntu".